libraries/platforms/dromajo-vcs/link.mk

```diff
@@ -122,8 +122,12 @@ $(BSG_PLATFORM_PATH)/test.riscv:
 $(BSG_PLATFORM_PATH)/libbsg_manycore_platform.so.1.0: $(DROMAJO_OBJECTS)
 $(BSG_PLATFORM_PATH)/libbsg_manycore_platform.so.1.0: $(PLATFORM_OBJECTS)
 $(BSG_PLATFORM_PATH)/libbsg_manycore_platform.so.1.0: LDFLAGS := -fPIC
+$(BSG_PLATFORM_PATH)/libbsg_manycore_platform.so.1.0: LDFLAGS += -L$(LIBRARIES_PATH)/features/dma/simulation -Wl,-rpath=$(LIBRARIES_PATH)/features/dma/simulation -ldmamem
+$(BSG_PLATFORM_PATH)/libbsg_manycore_platform.so.1.0: $(LIBRARIES_PATH)/features/dma/simulation/libdmamem.so
```

Comment:
nit - there's some code duplication here.

Reply:
I don't think so. The target name has been renamed here. The default is called `libbsg_manycore_runtime.so.1.0` but this is `libbsg_manycore_platform.so.1.0`. The LDFLAGS adds on to the `libbsg_manycore_runtime.so.1.0` target which I don't want since that is now a library compiled for BP.

Reply:
hm, ok.

Reply:
So is there any preferred suggestion to change it to?

Reply:
This is the platform for the C++ api that sends packets back and forth?

Reply:
This is the shared object that contains Dromajo and the C/C++ API that sends packets back and forth

Reply:
I think this name is good for now. Not sure what I'd change it to. I worry that there's a naming conflict with the runtime library, but not enough to change it at the moment

Reply:
A better place for this is in library.mk, I think. Where you define PLATFORM and DROMAJO OBJECTS.

Reply:
Done.